_legacy/.htaccess

```diff
@@ -1,31 +1,31 @@
 
 # Keyboard search
-# legacy /keyboards?q=... to /keyboards/index.php
-RedirectMatch "/_legacy/keyboards" "/keyboards/index.php"
+# legacy /keyboards?q=... to keyboards/index.php
+RewriteRule "^keyboards$" "/_legacy/keyboards/index.php" [END]
 
-# legacy /keyboards/ to /keyboards
-RedirectMatch 301 "/_legacy/keyboards/" "/keyboards"
+# legacy /keyboards/ to keyboards
+RewriteRule "^keyboards/$" "/_legacy/keyboards" [R=301,END,QSA]
 
-# legacy /keyboards/languages to /keyboards/index.php
-RedirectMatch "/_legacy/keyboards/languages/(.*)" "/keyboards/index.php?q=l:id:$1"
+# legacy /keyboards/languages to keyboards/index.php
+RewriteRule "^keyboards/languages/(.*)" "/_legacy/keyboards/index.php?q=l:id:$1" [END]
 
-# "legacy /keyboards/download to /keyboards/download.php"
-RedirectMatch "/_legacy/keyboards/download(\.php)?" "/keyboards/download.php"
+# "legacy /keyboards/download to keyboards/download.php
+RewriteRule "^keyboards/download(\.php)?" "/_legacy/keyboards/download.php" [END,QSA]
 
-# "legacy /keyboards/legacy to /keyboards/keyboard.php"
-RedirectMatch "/_legacy/keyboards/legacy/(.*)" "/keyboards/keyboard.php?legacy=$1"
+# "legacy /keyboards/legacy to keyboards/keyboard.php
+RewriteRule "^keyboards/legacy/(.*)" "/_legacy/keyboards/keyboard.php?legacy=$1" [END]
 
-# "legacy /keyboards/countries to /keyboards/index.php"
-RedirectMatch "/_legacy/keyboards/countries/(.*)" "/keyboards/index.php?q=c:id:$1"
+# "legacy /keyboards/countries to keyboards/index.php
+RewriteRule "^keyboards/countries/(.*)" "/_legacy/keyboards/index.php?q=c:id:$1" [END]
 
-# "legacy /keyboards/{name}/share to /keyboards/share.php"
-RedirectMatch "/_legacy/keyboards/([^/]+)/share" "/keyboards/share.php?id=$1"
+# "legacy /keyboards/{name}/share to keyboards/share.php
+RewriteRule "^keyboards/([^/]+)/share" "/_legacy/keyboards/share.php?id=$1" [END]
 
-# "legacy /keyboards/{name}.json to /keyboards/keyboard.json.php"
-RedirectMatch "/_legacy/keyboards/(.*)\.json$" "/keyboards/keyboard.json.php?id=$1"
+# "legacy /keyboards/{name}.json to keyboards/keyboard.json.php
+RewriteRule "^keyboards/(.*)\.json$" "/_legacy/keyboards/keyboard.json.php?id=$1" [END]
 
-# "legacy /keyboards/{name} to /keyboards/keyboard.php"
-RedirectMatch "/_legacy/keyboards/(.*)" "/keyboards/keyboard.php?id=$1"
+# "legacy /keyboards/{name} to keyboards/keyboard.php
+RewriteRule "^keyboards/(.*)" "/_legacy/keyboards/keyboard.php?id=$1" [END]
 
-# "legacy /keyboard/{name} to /keyboards/..." stopProcessing="true">
-RedirectMatch "/_legacy/keyboard/(.*)$" "/keyboards/$1"
+# "legacy /keyboard/{name} to keyboards/...
+RedirectMatch "/_legacy/keyboard/(.*)$" "/_legacy/keyboards/$1"
```

Comment:
These didn't seem to match the original comments, but it's what I had to do to get the links to match what's on the live site (currently using https://github.com/keymanapp/keyman.com/blob/master/_legacy/web.config)